packages/theme/components/RelatedProducts.vue

```diff
@@ -70,13 +77,19 @@ import {
   useWishlist
 } from '@vue-storefront/bigcommerce';
 import { Product } from '@vue-storefront/bigcommerce-api';
-import { defineComponent, PropType } from '@nuxtjs/composition-api';
+import {
+  defineComponent,
+  onBeforeUnmount,
+  PropType,
+  ref
+} from '@nuxtjs/composition-api';
 import { useProductData } from '../composables/useProductData';
 import { useWishlistData } from '../composables/useWishlistData';
 
 export default defineComponent({
   name: 'RelatedProducts',
   setup() {
+    const isMobile = ref(mapMobileObserver().isMobile.get());
```

Comment:
And here also `computed`
```suggestion
    const isMobile = computed(() => mapMobileObserver().isMobile.get());
```